cluster/gce/gci/configure-helper.sh

```diff
@@ -80,6 +82,21 @@ function get-calico-typha-cpu {
   echo "${typha_cpu}"
 }
 
+# Prepare node labels specifically for master or node.
+function prepare-node-labels {
```

Comment:
Copied unsolved comment from https://github.com/kubernetes/kubernetes/pull/51227#discussion_r135132176:
> move this to config-default.sh and config-test.sh.

I haven't figure out how yet. I thought config-default.sh and config-test.sh happen before we actually know it is on master or node (KUBERNETES_MASTER is not set yet)?

@mikedanese